test-framework/core/src/main/java/org/keycloak/testframework/server/AbstractKeycloakServerSupplier.java

```diff
@@ -55,6 +58,11 @@ public KeycloakServer getValue(InstanceContext<KeycloakServer, KeycloakIntegrati
             getLogger().debugv("Startup command and options: \n\t{0}", String.join("\n\t", command.toArgs()));
         }
 
+        if (command.tlsEnabled()) {
+            ManagedCertificates managedCert = instanceContext.getDependency(ManagedCertificates.class);
+            command.option("https-key-store-file", managedCert.getKeycloakServerKeyStorePath().toString());
```

Comment:
Just make ManagedCertificates return a string instead of a Path, since that's what we need.